### ISSUE TITLE
question about efficientnet

### ISSUE THREAD

**Issue by CoinCheung** [tensorflow/tpu#623]
Hi, 

Thanks for bring this great work to the community. I am studying the code. If I did not misunderstand the code, by simply run `python main.py`, the program will train efficient-b0 without mixup or autoaugment/randomaugment. Is this the configuration of the experiment in the paper where efficient-b0 got acc_top_1=76.3 and acc_top_5=93.2  ?

**Comment by mingxingtan** [comment]
Hi @CoinCheung , we have updated the arxiv paper to use open-source results: https://arxiv.org/pdf/1905.11946.pdf

The original 76.3% accuracy is based our internal code, which has some obvious implementation bugs.  The open source version is always 77.3% (with auto-augment), or 76.8% (with no auto-augment), as shown here: https://github.com/tensorflow/tpu/blob/master/models/official/efficientnet/README.md

**Comment by CoinCheung** [comment]
Did the experiments use mixup ?

**Comment by mingxingtan** [comment]
No, we don't use mixup.
